proto/options.proto

```diff
@@ -361,6 +361,11 @@ message Options {
     invariant = 3;
   }
 
+  enum IsochroneType {
+    forward = 0;
+    reverse = 1;
+  }
```

Comment:
at first i was thinking that we should make this more generic so we could use it elsewhere but now ive convinced myself we should make this simpler. specifically we should just make it a boolean.

```suggestion
```